modules/subscription/data.tf

```diff
@@ -0,0 +1,13 @@
+data "azurerm_client_config" "current" {}
+
+data "azuread_group" "ops_mgmt" {
+  display_name = "DTS Operations (env:mgmt)"
+}
+
+data "azuread_group" "aks_global_admin" {
+  display_name = "dcd_group_aks_admin_global_v2"
```

Comment:
I don't think we use this anymore, but if you remember / have time could you check after the PR is merged and clean up if so.